araproc/analysis/correlation.py

```diff
@@ -0,0 +1,114 @@
+import ROOT
+import numpy as np
+import math
+import os
+
+# HELP HERE: constants that we might want to store elsewhere, or can I retrieve them somehow?
```

Comment:
Maybe this is what you are looking for? Not sure if this has a standard reco as part of it already, but if it does you can use that I think. https://github.com/ara-software/AraProc/blob/a758550836aaafa473e05d6d976ee0bf1122dcdf/araproc/analysis/standard_reco.py#L65